examples_tests/61.BlitFilterTest/blit/formats/r11f_g11f_b10f.glsl

```diff
@@ -1,68 +1,49 @@
 #ifndef _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
 #define _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
 
-uint to11bitFloat(in float _f32)
+uint nbl_glsl_encode_ufloat(in float _f32, in uint mantissaMask, in uint mantissaBits)
```

Comment:
dont ask for two parameters that are redundant, work out the mask from the bitcount

Reply:
Resolved.